fulp_modules/features/halloween/2021/shovel_knight_outfits/propeller_knight.dm

```diff
@@ -0,0 +1,38 @@
+/**
+ * Propeller Knight
+ * From: Shovel knight
+ * By: Sheets
+ */
+
+///suit (uses hood to toggle helmet)
+
+/obj/item/clothing/suit/hooded/costume_2021/propeller_suit
+	name = "Propeller knight's suit"
```

Comment:
I dont know if this is a name, if it isnt, this isnt supposed to start capitalized, same for the others.